spec/integration/login_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Login', type: :system do
+  before do
+    user = User.new(name: 'Jamie', email: 'jamie@gmail.com', password: 'jamiepass',
+                    password_confirmation: 'jamiepass')
+    user.skip_confirmation!
+    user.save!
+  end
+
+  scenario 'should be successful' do
+    visit new_user_session_path
+    fill_in 'Email', with: 'jamie@gmail.com'
+    fill_in 'Password', with: 'jamiepass'
+    click_button 'Log in'
+    expect(page).to have_content('Signed in successfully.')
+  end
+
+  scenario 'should fail' do
+    visit new_user_session_path
+    fill_in 'Email', with: 'jamie@gmial.com'
+    fill_in 'Password', with: 'jamie'
+    click_button 'Log in'
+    expect(page).to have_content('Invalid Email or password.')
+  end
+
+  scenario 'should see Login' do
+    visit root_path
```

Comment:
- [ ] Your tests are good. I am really impressed 👏 However, some of them are failing That is why I would suggest that you fix them 🙏 
![image](https://user-images.githubusercontent.com/90222110/230642473-69530506-e6e9-40c4-a11d-f1b98a5e2e37.png)

Reply:
Addressed